app/src/main/res/layout/activity_passenger_list.xml

```diff
@@ -6,4 +6,136 @@
+        android:layout_height="wrap_content"
+        android:text="BUSIFY"
+        android:paddingStart="25dp"
+        android:textSize="48sp"
+        android:textStyle="bold"
+        android:background="#FBBC05"
+        android:textColor="@color/white"
+        />
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/search"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical"
+        app:layout_constraintTop_toBottomOf="@+id/logo"
+        app:cardCornerRadius="50dp"
+        android:layout_marginStart="25dp"
+        android:layout_marginEnd="25dp"
+        android:layout_marginTop="18dp"
+        >
+        <androidx.appcompat.widget.LinearLayoutCompat
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="center_vertical"
+            android:padding="10dp"
+            >
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:src="@drawable/search_icon"
+                android:layout_marginStart="5dp"
+                />
+            <androidx.appcompat.widget.AppCompatEditText
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@color/white"
+                android:hint="Search"
+                android:textSize="25sp"
+                android:layout_marginStart="30dp"
+                />
+
+        </androidx.appcompat.widget.LinearLayoutCompat>
+    </androidx.cardview.widget.CardView>
+    <androidx.cardview.widget.CardView
+        android:layout_width = "match_parent"
+        android:layout_height = "wrap_content"
+        app:layout_constraintTop_toBottomOf="@id/search"
+        android:layout_marginTop = "35dp"
+        android:layout_marginStart = "25dp"
+        android:layout_marginEnd = "25dp"
+        app:cardCornerRadius = "15dp"
+        app:cardElevation="10dp"
+        android:id="@+id/bus"
+        >
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width = "match_parent"
+            android:layout_height = "wrap_content"
+            android:padding = "20dp"
+            >
+
+            <androidx.appcompat.widget.LinearLayoutCompat
+                android:layout_width = "match_parent"
+                android:layout_height = "wrap_content"
+                app:layout_constraintTop_toTopOf="parent"
+                android:id="@+id/time"
+                android:gravity="center_vertical"
+                app:layout_constraintStart_toStartOf="parent"
+                >
+
+                <androidx.appcompat.widget.AppCompatImageView
+                    android:id = "@+id/imageView"
+                    android:layout_width = "wrap_content"
+                    android:layout_height = "wrap_content"
```

Comment:
Don't need to write the code for this view again.
It is similar to view, merged in this issue #3,
Use include to reuse the layouts, you can take reference from [here](https://developer.android.com/develop/ui/views/layout/improving-layouts/reusing-layouts)